### ISSUE TITLE
AWP

### ISSUE THREAD

**Issue by riow1983** [riow1983/nbme-score-clinical-patient-notes#4]
Wu, Xia, and Wang (2020), Adversarial Weight Perturbation Helps Robust Generalization https://arxiv.org/abs/2004.05884

Authors' implementation: https://github.com/csdongxian/AWP

**Comment by riow1983** [comment]
[What is threat model?]
> An adversarial threat model defines a set of perturbations that may be made to an image in
order to produce an adversarial example. Common threat models include L2 and L∞ threat models,
which constrain adversarial examples to be close to the original image in L2 or L∞ distances.

https://arxiv.org/pdf/2006.12655.pdf

**Comment by riow1983** [comment]
ATまとめ

https://www.slideshare.net/ssuserc45ddf/adversarial-trainingpptx

**Comment by riow1983** [comment]
Fast AWP (Kaggle notebook)
https://www.kaggle.com/code/junkoda/fast-awp

Video (Chinese)
https://www.bilibili.com/s/video/BV1pt4y1Y7uU

**Comment by riow1983** [comment]
PACベイズの基礎についてまとめる
https://qiita.com/student-i/items/341f1fd720d4b6b26d7b

**Comment by riow1983** [comment]
Kaggle notebook:
https://www.kaggle.com/code/wht1996/feedback-nn-train/notebook

**Comment by riow1983** [comment]
社内発表完了.

**Comment by riow1983** [comment]
***
[SGDとシグナム関数]
The Geometry of Sign Gradient Descent
https://arxiv.org/pdf/2002.08056.pdf

***
[確率とバウンドについて]
ヘフディングの不等式(Hoeffding's inequality)と諸々の確率の評価の不等式
https://ludu-vorton.hatenablog.com/entry/2019/06/06/073000

CS229 Supplemental Lecture notes Hoeffding’s inequality
http://cs229.stanford.edu/extra-notes/hoeffding.pdf

***
[VC次元とDLの汎化誤差見積もりについて]
過学習に理論的に迫る
https://qiita.com/kuroitu/items/e695907d9a2d28cc0cf9

(Not) Bounding the True Error
https://papers.nips.cc/paper/2001/file/98c7242894844ecd6ec94af67ac8247d-Paper.pdf

Exploring Generalization in Deep Learning
https://papers.nips.cc/paper/2017/file/10ce03a1ed01077e3e289f3e53c72813-Paper.pdf

Computing Nonvacuous Generalization Bounds for Deep (Stochastic) Neural Networks with Many More Parameters than Training Data
https://arxiv.org/pdf/1703.11008.pdf

***
[PAC Bayes]
An Introduction to PAC-Bayes
https://www.youtube.com/watch?v=t5GBuBD0ibc

Some PAC-Bayesian Theorems (McAllester's theorem)
https://link.springer.com/content/pdf/10.1023/A:1007618624809.pdf
